### DISCUSSION TITLE
Statsd pod restarting due to liveness probe failure

### DISCUSSION THREAD

**Discussion by hasland** [apache/airflow#36978] [General]
Hello everyone,
We are facing an issue with statsd pod: it continues to restart because of a liveness probe failure. Debugging, we've found the following on _statsd-pod:9092/metrics_ endpoint:

```An error has occurred while serving metrics:
collected metric named "airflow_dag_processing_last_duration_sum" collides with previously collected summary named "airflow_dag_processing_last_duration"
```
 and a return code 500 that makes the liveness probe fail.

Has anyone faced this problem? Any thoughts? 

Environment:
Kubernetes v1.27.7, Airflow 2.8.1 with Python 11.

![image](https://github.com/apache/airflow/assets/18452061/fedcab08-a689-45ed-8ee7-16b9432601e9)

**Comment by potiuk** [comment]
I believe (but this is a guess  it's been solved in relatively recent versions of prometheus statsd-exporter. 

Looks like your error is descibed here: https://github.com/prometheus/statsd_exporter/issues/186

And seems to be fixed here: https://github.com/prometheus/statsd_exporter/pull/213 (a looong time ago). So I generally suggest first to update (not only statsd-exported but likely other images too) . It won't hurt

**Reply by hasland** [reply]
We tried with new versions of statsd (v0.25.0 for example) but the problem persists. Other images, like Airflow, Prometheus, are all updated to some recent version (prometheus v2.47.1, airflow 2.8.1)
Any other idea that we can try?

**Reply by potiuk** [reply]
Nope. I just googled it.
